website/src/lib/formatMenu.js

```diff
@@ -0,0 +1,26 @@
+// https://www.wpgraphql.com/docs/menus/#hierarchical-data
+/* eslint-disable */
+
+export default function formatMenu(
+  data = [],
+  {
+    idKey = 'id',
+    parentKey = 'parentId',
+    childrenKey = 'children',
+  } = {},
```

Comment:
Since each object keys have a default above, _I think_ the object as a whole doesn't need to fallback to empty object

Reply:
You maybe right, but I am trying to use the wpgraphql provided code exactly as is, not refactor.